l10n_fi_edicode/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: LGPL
```

Comment:
Sorry for the confusion. I just remembered that it'd be best not to write `README.rst` by hand, but instead let the OCA bot take care of generating it. Take a look at: https://github.com/OCA/queue/tree/13.0/queue_job and especially the `readme` folder.

Here's the script that generates the readme: https://github.com/OCA/maintainer-tools#readme-generator

Reply:
And also, the correct form in the manifest file should be `LGPL-3`.